appimage/build_appimage.sh

```diff
@@ -128,7 +137,7 @@ install_monkey_island_python_dependencies() {
   #       dependencies and should not be installed as a runtime requirement.
   cat "$requirements_island" | grep -Piv "virtualenv|pyinstaller" | sponge "$requirements_island"
 
-  ${python_cmd} -m pip install -r "${requirements_island}"  --ignore-installed --prefix /usr --root="$APPDIR" || handle_error
+  "$APPDIR"/AppRun -m pip install -r "${requirements_island}"  --ignore-installed || handle_error
 }
 
 download_monkey_agent_binaries() {
```

Comment:
It'll be easier to include into jenkins if we move artifacts between machines (scp) rather then uploading to some HTTP server. But this can stay for now